website/docs/r/alikafka_instance.html.markdown

```diff
@@ -74,7 +74,10 @@ The following arguments are supported:
 * `deploy_type` - (Required) The deployment type of the instance. **NOTE:** From version 1.161.0, this attribute supports to be updated. Valid values:
   - 4: eip/vpc instance
   - 5: vpc instance.
-* `io_max` - (Required) The max value of io of the instance. When modify this value, it only support adjust to a greater value.
+* `io_max` - (Optional, Computed) The max value of io of the instance. When modify this value, it only support adjust to a greater value.
+* `io_max_spec` - (Optional, Computed, Available in v1.201.0+) The traffic specification of the instance. We recommend that you configure this parameter.
+  - You must specify one of the `io_max` and `io_max_spec` parameters. We recommend that you specify the `io_max_spec` parameter.
```

Comment:
You should specify one of the `io_max` and `io_max_spec` parameters, and `io_max_spec` is recommended.